R/readIEA_PRICES.R

```diff
@@ -0,0 +1,54 @@
+#' readIEA_PRICES
+#'
+#' Read in energy prices from International Energy Agency.
+#'
+#' @param subtype Type of data that should be read.
+#' @return The read-in data into a magpie object.
+#'
+#' @author Anastasis Giannousakis, Fotis Sioutas
+#'
+#' @examples
+#' \dontrun{
+#' a <- readSource("IEA_PRICES", subtype = "ELECTR")
```

Comment:
object returned contains a unit value as column name? ("2015 USD/unit"), and obsolete columns
```> str(a)
A magpie object (package: magclass)
 @ .Data:  num [1:39591] NA NA NA NA NA NA NA NA NA NA ...
 $ dimnames:List of 3
  ..$ region                                     : chr [1:249] "ABW" "AFG" "AGO" "AIA" ...
  ..$ period                                     : chr [1:53] "y1970" "y1971" "y1972" "y1973" ...
  ..$ model.scenario.variable.unit.product.sector: chr [1:3] "(Missing).(Missing).(Missing).2015 USD/unit.ELECTR.RESID" "(Missing).(Missing).(Missing).2015 USD/unit.ELECTR.IND" "(Missing).(Missing).(Missing).2015 USD/unit.ELECTR.COMM"
```

Reply:
ok